package.json

```diff
@@ -93,6 +93,7 @@
   },
   "dependencies": {
     "@lvce-editor/ripgrep": "^1.6.0",
```

Comment:
please update the existing ink dependency in the cli package rather than adding this one.